resource_obmcs_objectstorage_object_test.go

```diff
@@ -41,9 +41,13 @@ func (s *ResourceObjectstorageObjectTestSuite) SetupTest() {
 	s.TimeCreated = baremetal.Time{Time: time.Now()}
 
 	s.Config = `
+		variable "accesstype" {
+			default = "ObjectRead"
+		}
 		resource "baremetal_objectstorage_bucket" "t" {
 			compartment_id = "${var.compartment_id}"
 			name = "bucketID"
+			access_type="${var.accesstype}"
```

Comment:
is there any reason not to just inline:
```hcl
access_type = "ObjectRead"
```

I also like to run my configs through `terraform fmt`, but clearly that wasn't the case when you came to the code.